plenum/test/pp_seq_no_restoration/test_last_sent_pp_store_helper.py

```diff
@@ -156,6 +156,7 @@ def test_try_restore_last_sent_pp_seq_no_if_invalid_stored(
     assert replica.H == 0 + tconf.LOG_SIZE
 
 
+@pytest.mark.skip(reason="INDY-1336: pp_seq_no not depend on view_no anymore")
```

Comment:
Should we rather modify the test instead of skipping it?

Reply:
ok